fosite-example/main.go

```diff
@@ -131,6 +131,11 @@ func tokenEndpoint(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	// Grant requested scopes
```

Comment:
In `ResourceOwnerPasswordCredentialsGrantHandler.PopulateTokenEndpointResponse()` 
I'm using `if requester.GetGrantedScopes().Has("offline")` to determine if they need a refresh token but it seems like this wasn't set anywhere. 
I compared it to the authorize endpoint which set the scopes like this.

The NewAccessRequest seems to already throw an error if a requested scope isn't allowed by a client so I thought this may be safe to do here.

What do you think?